README.md

```diff
@@ -73,6 +75,10 @@ Some details worth highlighting:
   - There is the `override` field (for now required) which defines whether this 
 entry _overrides_ the previous service of the same name (if any - missing is 
 okay), or merges with it.
+  - The optional `user` field allows starting a service with a different user
```

Comment:
This text is no longer true, as you have to pass a numeric user -id to 'user-id' not 'user'.

Reply:
Updated.